src/ethereum/crypto.py

```diff
@@ -20,6 +20,8 @@
 Hash32 = Bytes32
 Hash64 = Bytes64
 
+secp256k1n = 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEBAAEDCE6AF48A03BBFD25E8CD0364141
```

Comment:
Not sure exactly how to do the screaming snake case for this one, but I think I prefer uppercase constants.

```suggestion
SECP256K1N = 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEBAAEDCE6AF48A03BBFD25E8CD0364141
```

Reply:
done